app/assets/stylesheets/to_change_everything/quebecois/_font_overrides.scss

```diff
@@ -0,0 +1,88 @@
+@import "../common/font_styling";
+
+.title-container h1 {
```

Comment:
actually, i think most of these are copy/paste from the `common/font_styling`. let me de-dupe and reduce this further

Reply:
fixed